shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1662,7 +1662,7 @@ public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOE
             String segment = in.readUTF();
             String delegationToken = in.readUTF();
             TableEntries entries = (TableEntries) TableEntries.readFrom(in, in.available());
-            long tableSegmentOffset = (in.available() > 0 ) ? in.readLong() : NULL_TABLE_SEGMENT_OFFSET;
+            long tableSegmentOffset = (in.available() >= 0 ) ? in.readLong() : NULL_TABLE_SEGMENT_OFFSET;
```

Comment:
Note: I'm assuming that an offset of 0 is valid.

Reply:
An offset of 0 is valid, but why are you making this change? This checks for how many bytes are available for reading from the buffer stream.

Reply:
Touche.. Had a little bit of a brain fart. Will revert.